crates/red_knot_python_semantic/src/types.rs

```diff
@@ -276,6 +278,10 @@ impl<'db> Type<'db> {
                 Type::Unknown
             }
             Type::BooleanLiteral(_) => Type::Unknown,
+            Type::BytesLiteral(_) => {
+                // TODO check if bytes have members
```

Comment:
We will probably want to just defer to `Type::Instance(<bytes from typeshed>).member` here, you could update the TODO but it's not critical